dist/pythonlibs/riotctrl_shell/tests/test_gnrc.py

```diff
@@ -0,0 +1,117 @@
+# Copyright (C) 2020 Freie Universität Berlin
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+import logging
+import os
+import riotctrl_shell.gnrc
+import subprocess
+
+from common import init_ctrl
+
+
+logger = logging.getLogger(__name__)
+
+
+def tap_exists(tap="tap0"):
+    try:
+        subprocess.check_call(["ip", "link", "show", tap])
+    except subprocess.CalledProcessError:
+        logger.error("\033[1;31mAborting test! Please make sure there is "
+                     "a TAP interface named \033[1;37m{}\033[0;0m".format(tap))
+        return False
+    return True
+
+
+def test_ping6():
+    rc = init_ctrl()
+    assert tap_exists()
+    si = riotctrl_shell.gnrc.GNRCICMPv6Echo(rc)
+    res = si.ping6("::1", interval=100)
+    # mock just returns last input
```

Comment:
While the comment refers to the mock, since our shell echos the last command as well before the command output, this is also true for RIOT's shell (in case the mock is not used for some reason).